oci-pipeline-examples/oci-cascaded-pipelines/same-region/trigger-build-from-build/README.md

```diff
@@ -0,0 +1,59 @@
+# OCI DevOps Cascaded Pipelines
+
+## Trigger Build Pipeline from Build Pipeline
+
+### Prerequisites
+* Two build pipelines
+
+#### 1. Deploy Function
+**Step 1:**
+Clone the repository code.
+```
+$ git clone https://github.com/oracle-cloud-infra/oci-devops-cascaded-pipelines.git
+```
+
+**Step 2:**
+Change to function directory.
+```
+$ cd oci-devops-cascaded-pipelines/same-region/trigger-build-from-build/node-function
+```
+
+**Step 3:**
+In `func.js`, update `buildPipelineId` and `displayNamePrefixForNewRun` with appropriate values as below.
+
+```
+const buildPipelineId = "<target-build-pipeline-ocid>"
+const displayNamePrefixForNewRun = "AutoTriggeredCascadedPipeline_";
+```
```

Comment:
Move the variable updates to func.yaml as a config and read them as os env from the code - sample -  https://github.com/RahulMR42/oci-helmdeploy-with-artifactrepo-and-functions/blob/main/func.yaml#L8